language-server/compile-language-server.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+GITHUB_REPO_URL="https://github.com/vscode-langservers/vscode-json-languageserver"
+GITHUB_REPO_NAME=$(echo "${GITHUB_REPO_URL}" | command grep -oE '[^/]*$')
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/${GITHUB_REPO_NAME}"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# -------- #
+# clean up #
+# -------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf \
+    "${DIST_DIR}" \
+    "package.json" "package-lock.json"
+
+popd || exit
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+echo 'Enter commit SHA, branch or tag (for example 2.1.0) to build'
+read -rp 'SHA, branch or tag (default: master): ' ref
+
+# use the "master" branch by default
+if [ "${ref}" = "" ]; then
+    ref="master"
+fi
+
+temp_zip="src-${ref}.zip"
+curl -L "${GITHUB_REPO_URL}/archive/${ref}.zip" -o "${temp_zip}"
+unzip -z "${temp_zip}" > update-info.log
```

Comment:
Can you try with `-zb` option here instead of `-z`?
The file has Windows line endings currently which makes the file change depending on which system it runs.

Reply:
`-zb` seems has no special effect on my Windows machine. I think https://github.com/sublimelsp/LSP-json/pull/37/commits/45f81c521c11a9f4f302e9a8ec40c314639d17b9#diff-f12eb39acb360e835c9399c7b0fbda8aR41 should work for everyone.